Libraries/LibGUI/AboutDialog.cpp

```diff
@@ -42,6 +42,9 @@ AboutDialog::AboutDialog(const StringView& name, const Gfx::Bitmap* icon, Window
     set_title(String::format("About %s", m_name.characters()));
     set_resizable(false);
 
+    if (parent_window)
+        set_icon(parent_window->icon());
```

Comment:
On second thought this could as well go into `GUI::Dialog`. That way `MessageBox` and friends would also inherit their parents' window icons.

Reply:
It works quite well with them having the default icons too IMO.